common/djangoapps/util/enterprise_helpers.py

```diff
@@ -0,0 +1,122 @@
+from django.conf import settings
+from django.utils.translation import ugettext as _
+
+try:
+    from enterprise.models import EnterpriseCustomer
+    from enterprise.tpa_pipeline import (
+        active_provider_requests_data_sharing,
+        active_provider_enforces_data_sharing,
+        get_enterprise_customer_for_request,
+    )
+
+except ImportError:
+    pass
+
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+
+
+def enterprise_enabled():
+    """
+    Determines whether the Enterprise app is installed
+    """
+    return 'enterprise' in settings.INSTALLED_APPS
```

Comment:
@asadiqbal08, I lean towards not requiring a feature flag; if the app is installed, it's enabled. If it's not, it's disabled. Adding a flag adds a level of complexity that we need to maintain in terms of _actually_ disabling features when that flag is present.